### PR TITLE
[CHEC-1030] Add Privacy Manifest

### PR DESCRIPTION
**Pull request by uziwuzzy**
1. set ios deployment target to ios 12
2. add privacy manifest for all integration (manual, spm, cocoapods)

MidtransCoreKit:
1. SPM:
Screenshot 2024-04-22 at 14 28 18

2. Cocoapods:
Screenshot 2024-04-22 at 14 30 19

3. Manual:

Screenshot 2024-04-23 at 07 31 55

MidtransKit:
1. SPM:
Screenshot 2024-04-22 at 14 25 45

2. Cocoapods:
Screenshot 2024-04-22 at 14 35 48

3. Manual
Screenshot 2024-04-22 at 14 43 45

### CODE REVIEW

File: MidtransCoreKit.podspec [hunk 1]
```diff
@@ -7,7 +7,9 @@ s.homepage         = "https://veritrans.co.id/"
 s.license          = 'MIT'
 s.author           = { "veritrans" => "dev@veritrans.co.id" }
 s.source           = { :git => 'https://github.com/veritrans/Veritrans-ios-sdk.git', :tag => s.version}
-s.platform     = :ios, '9.0'
+s.platform     = :ios, '12.0'
```
Comment (fixurlinter) [inline MidtransCoreKit.podspec:10]: whats the reason to update  `platform/ios.deployment_target` to 12?

File: MidtransCoreKit/MidtransCoreKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1754,7 +1758,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/MidtransCoreKit/RG",
 				);
-				MARKETING_VERSION = 1.24.1;
+				MARKETING_VERSION = 1.24.2;
```
Comment (fixurlinter) [inline MidtransCoreKit/MidtransCoreKit.xcodeproj/project.pbxproj:1761]: CMIIW `MARKETING_VERSION` is APP_VERSION?

File: MidtransCoreKit/PrivacyInfo.xcprivacy [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```
Comment (fixurlinter) [inline MidtransCoreKit/PrivacyInfo.xcprivacy:3]: could you explain, how to generate this file?

File: MidtransCoreKit.podspec (hunk 1 above)
Comment (uziwuzzy) [reply MidtransCoreKit.podspec:10]: hmm i think I forgot to revert this commit on the podspec, will revert this.

File: MidtransCoreKit/PrivacyInfo.xcprivacy (hunk 3 above)
Comment (uziwuzzy) [reply MidtransCoreKit/PrivacyInfo.xcprivacy:3]: you can go to xcode, and go to File → New → File , scroll down a little bit to the Resource section and select App privacy

**Comment by fixurlinter** [comment]
LGTM

File: MidtransCoreKit.podspec [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Pod::Spec.new do |s|
+ Pod::Spec.new do |s|
```
Comment (fwilson7) [inline MidtransCoreKit.podspec:1]:
```suggestion
Pod::Spec.new do |s|
```

**Review by fwilson7 — APPROVED** [review]
LGTM

**Pull request merged**